pyproject.toml

```
@@ -10,9 +10,10 @@ repository = "https://github.com/Ben-Hu/metabasic"
 include = ["metabasic/py.typed"]
 
 [tool.poetry.dependencies]
-python = ">=3.6"
+python = ">=3.7"
```

`
```suggestion
python = ">=3.6.1"
```

Should be sufficient here to satisfy the Pandas python version requirement while keeping compatibility with python 3.6

---

done!